src/applications/ask-va/components/FileUpload.jsx

```diff
@@ -2,28 +2,69 @@ import { VaFileInput } from '@department-of-veterans-affairs/component-library/d
 import PropTypes from 'prop-types';
 import React, { useState } from 'react';
 
+const idList = numberOfIDs => {
+  const ids = [];
+  for (let i = 0; i < numberOfIDs; i++) {
+    ids.push(`askVA_upload_${i}`);
+  }
+  return ids;
+};
+
 const FileUpload = props => {
   const {
     acceptFileTypes = '.pdf,.jpeg,.png',
     buttonText = 'Upload file',
     error,
-    label = 'Upload your file',
-    header = 'Upload your files',
-    hint = null,
+    label = 'Select files to upload',
+    header = 'Select files to upload',
+    hint = 'You can upload a .pdf, .jpeg, or .png file.',
```

Comment:
![image](https://github.com/user-attachments/assets/dec8067d-7a92-41ba-8b06-2460a5bced29)


I think we need to update the text here per figma

Reply:
done